redbot/cogs/audio/core/commands/localtracks.py

```diff
@@ -47,11 +57,26 @@ async def command_local_folder(self, ctx: commands.Context, *, folder: str = Non
             query = Query.process_input(
                 _dir, self.local_folder_current_path, search_subfolders=True
             )
-            await self._local_play_all(ctx, query, from_search=False if not folder else True)
+            await self._local_play_all(ctx, query, from_search=bool(folder))
 
     @command_local.command(name="play")
     async def command_local_play(self, ctx: commands.Context):
-        """Play a local track."""
+        """Play a local track.
+
+        To set up your localtracks folders, please see the documentation at: `placeholder`
+        To play a local track, either use the menu to choose a track or enter in the
+        localtracks path directly with the play command.
+        To play an entire local folder, use [p]help local folder for instructions.
+
+        Examples:
+        [p]local play
+        Open a menu to pick a file.
+
+        [p]play localtracks/album_folder/song_name.mp3
```

Comment:
Given that the known majority of users are on windows, please use windows paths here.